src/EFCore.PG.Trgm/Extensions/NpgsqlTrgmDbContextOptionsBuilderExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.EntityFrameworkCore.Infrastructure;
+
```

Comment:
nit: We don't split usings with empty lines.

Reply:
24ff45d1edc00f0f4748be34ee18e0c0eee25f3e fixes this.